pages/pipelines/pipelines.md.erb

```diff
@@ -62,4 +62,35 @@ Your pipeline’s name, description, its repository, and your default branch are
 
 The display settings can be found in the `Pipeline Settings` section. Adding a description for your pipeline is optional, but name, repository, and default branch are all required. Descriptions have full emoji support! 🙌:llama:💯  
 
+## My Builds
+
+In contrast (something softer, like conjunction) with the high level pipeline information on the Pipelines page, the My Builds dropdown menu provides a personalised list of your most recently triggered builds.   
+
+### Navigation item
+
+Living in the navigation bar, the My Builds menu is directly accessible from every page in the Buildkite web application. The count that appears in the My Builds navigation bar item tells you how many of your builds are running or waiting to run.
```

Comment:
Uhh, not sure. I think probably just Buildkite... I was thinking about how sometimes we need to make the distinction between stuff you can access with the buildkite-agent command and the stuff you can get to in the UI. But that isn't this section, so I think just 'Buildkite' would be good here.